stl/inc/tuple

```diff
@@ -866,10 +866,40 @@ _NODISCARD constexpr _Ty&& get(array<_Ty, _Size>&& _Arr) noexcept;
 template <size_t _Idx, class _Ty, size_t _Size>
 _NODISCARD constexpr const _Ty&& get(const array<_Ty, _Size>&& _Arr) noexcept;
 
+// STRUCT TEMPLATE _Repeat_type_in_tuple
+template <class _Ty, class... _Meta>
+struct _Repeat_type_in_tuple {
+    static_assert(_Always_false<_Ty>, "Requires index_sequence as second template argument.");
+};
+
+template <class _Ty, size_t... _Ix>
+struct _Repeat_type_in_tuple<_Ty, index_sequence<_Ix...>> {
+    using type = tuple<typename remove_reference_t<decltype(_Ix, declval<_Identity<_Ty>>())>::type...>;
+};
+
+// STRUCT TEMPLATE _Is_tuple_like
+template <class _Ty, class = int>
+struct _Is_tuple_like : false_type {};
+
+template <class _Ty>
+struct _Is_tuple_like<_Ty, decltype((void)tuple_size<_Ty>::size, 0)> : true_type {}; 

```

Comment:
This partial specialization will never be matched; did you mean to `class = void` on 881?

Reply:
```suggestion
struct _Is_tuple_like<_Ty, decltype((void)tuple_size<_Ty>::value, 0)> : true_type {};
```

Reply:
I missed that. Now `void_t` is used